templates/generate_tuning_curve.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
<p>Add a preface to this notebook saying that we've generated standard tuning curves which will work across instruments, so unless there is a specific reason the user needs a custom curve, this notebook shouldn't be necessary, use the default ones, etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>